### PR TITLE
Sugar Gallery

### PR DESCRIPTION
**Pull request by inkyfingers**
A gallery might be used differently in future, but this pull request introduces 3 slide show presentations aimed at New Sugar learners.

The presentations can be previewed and tested from http://inkyfingers.github.io/gallery

### CODE REVIEW

**Comment by davelab6** [comment]
This is great!

- the HTML intro page could be better as a markdown file
- the font choices could be improved, perhaps Varela Rounded
- the site navigation could be improved
- `/gallery` doesn't seem ideal for l10n, I suggest `/en_US/gallery` :)

File: index.html [hunk 1]
```diff
@@ -60,6 +60,13 @@ <h1 class="from-toolbar">This is Sugar</h1>
   </ul>
 </div>
 
+<div>
+  <a href="/gallery/index"><div class="star-link lime">
```
Comment (davelab6) [inline index.html]: This can be `<a href="gallery">`

**Comment by davelab6** [comment]
Why are you using remark.js when this is hosted in Jeykll? :)

**Comment by inkyfingers** [comment]
Great, thanks @davelab6 for the positive and useful feedback :)

Fundamentals first,

>Why are you using remark.js when this is hosted in Jeykll? :)

I wrote this at a time when www.sugarlabs.org was **not** hosted in Jekyll, then we had this conversation

http://lists.sugarlabs.org/archive/iaep/2016-April/017953.html

**Comment by davelab6** [comment]
>  then we had this conversation

Okay :) I suggest I will make a PR to your repo to show you how I would do it, and then you can update this PR :)

**Comment by inkyfingers** [comment]
Thanks, that would be grand, but having merged upstream, my local machine cannot `jekyll serve` or `jekyll serve --incremental;` right now!
I see this change,
`+gem 'github-pages', group: :jekyll_plugins`
 but cannot figure what I have to do next.
The compaint is 
`Could not find gem 'github-pages' in any of the gem sources listed in your Gemfile or available on this machine.`

**Comment by davelab6** [comment]
What version of Ruby are you running?

**Comment by davelab6** [comment]
The `Gemfile` was added in https://github.com/sugarlabs/www-sugarlabs/commit/315ddde956770090f4c02dc02ecd773804087831 along with instructions; try running

> bundle install

**Comment by inkyfingers** [comment]
`$ ruby -v
ruby 2.3.0p0 (2015-12-25) [x86_64-linux-gnu]`

> Gemfile was added in 315ddde along with instructions 

Seen,

`$ bundle install;
Your Gemfile has no gem server sources. If you need gems that are not already on your machine,
add a line like this to your Gemfile:
source 'https://rubygems.org'
Could not find gem 'github-pages' in any of the gem sources listed in your Gemfile or available on this machine.`

**Comment by davelab6** [comment]
I get:

```
$ bundle install
Resolving dependencies...
Rubygems 2.0.14.1 is not threadsafe, so your gems will be installed one at a time. Upgrade to Rubygems 2.1.0 or higher to enable parallel gem installation.
Using i18n 0.7.0
Using json 1.8.3
Using minitest 5.8.4
Using thread_safe 0.3.5
Using addressable 2.4.0
Using coffee-script-source 1.10.0
Using execjs 2.6.0
Using colorator 0.1
Using ffi 1.9.10
Using multipart-post 2.0.0
Using gemoji 2.1.0
Using net-dns 0.8.0
Using public_suffix 1.5.3
Using sass 3.4.22
Using rb-fsevent 0.9.7
Using kramdown 1.10.0
Using liquid 3.0.6
Using mercenary 0.3.6
Using rouge 1.10.1
Using safe_yaml 1.0.4
Using jekyll-feed 0.5.1
Using mini_portile2 2.0.0
Using jekyll-paginate 1.1.0
Using jekyll-sitemap 0.10.0
Using terminal-table 1.5.2
Using bundler 1.12.3
Using tzinfo 1.2.2
Using coffee-script 2.4.1
Using ethon 0.9.0
Using rb-inotify 0.9.7
Using faraday 0.9.2
Using jekyll-sass-converter 1.3.0
Using nokogiri 1.6.7.2
Using activesupport 4.2.6
Using jekyll-coffeescript 1.0.1
Using typhoeus 0.8.0
Using listen 3.0.6
Using sawyer 0.7.0
Using html-pipeline 2.4.1
Using jekyll-watch 1.4.0
Using octokit 4.3.0
Using jekyll 3.0.5
Using github-pages-health-check 1.1.0
Using jekyll-gist 1.4.0
Using jekyll-github-metadata 1.11.1
Using jekyll-mentions 1.1.2
Using jekyll-redirect-from 0.10.0
Using jekyll-seo-tag 1.4.0
Using jemoji 0.6.2
Using github-pages 78
Bundle complete! 1 Gemfile dependency, 50 gems now installed.
Use `bundle show [gemname]` to see where a bundled gem is installed.
$
```

**Comment by davelab6** [comment]
Ok I pushed 251f89c this might help

**Comment by inkyfingers** [comment]
So I had already added to gemfile

`source 'https://rubygems.org'`

Primary error was, I ran
`$ bundle install;`
should be
`$ bundle install`

zlib is missing; necessary for building libxml2
...
https://github.com/flapjack/omnibus-flapjack/issues/72
...
planning to:
`sudo apt-get install zlib1g-dev`

... I think :) - thanks so much for your help!

**Comment by davelab6** [comment]
Weird, the semicolon shouldn't matter. What shell are you using?

**Comment by davelab6** [comment]
So you succeeded with `sudo apt-get install zlib1g-dev`?

**Comment by inkyfingers** [comment]
Yes, I got a successful "Post-install message from github-pages:"

... but

$ jekyll serve --incremental;
WARN: Unresolved specs during Gem::Specification.reset:
      jekyll-watch (~> 1.1)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
Configuration file: /home/iain/www-sugarlabs/_config.yml
  Dependency Error: Yikes! It looks like you don't have kramdown or one of its dependencies installed. In order to use Jekyll as currently configured, you'll need to install this gem. The full error message from Ruby is: 'cannot load such file -- kramdown' If you run into trouble, you can find helpful resources at http://jekyllrb.com/help/! 
jekyll 3.1.2 | Error:  kramdown

**Comment by inkyfingers** [comment]
$ bundle list
Gems included by the bundle:
  * activesupport (4.2.6)
  * addressable (2.4.0)
  * bundler (1.11.2)
  * coffee-script (2.4.1)
  * coffee-script-source (1.10.0)
  * colorator (0.1)
  * ethon (0.9.0)
  * execjs (2.6.0)
  * faraday (0.9.2)
  * ffi (1.9.10)
  * gemoji (2.1.0)
  * github-pages (78)
  * github-pages-health-check (1.1.0)
  * html-pipeline (2.4.1)
  * i18n (0.7.0)
  * jekyll (3.0.5)
  * jekyll-coffeescript (1.0.1)
  * jekyll-feed (0.5.1)
  * jekyll-gist (1.4.0)
  * jekyll-github-metadata (1.11.1)
  * jekyll-mentions (1.1.2)
  * jekyll-paginate (1.1.0)
  * jekyll-redirect-from (0.10.0)
  * jekyll-sass-converter (1.3.0)
  * jekyll-seo-tag (1.4.0)
  * jekyll-sitemap (0.10.0)
  * jekyll-watch (1.4.0)
  * jemoji (0.6.2)
  * json (1.8.3)
  * kramdown (1.10.0)
  * liquid (3.0.6)
  * listen (3.0.6)
  * mercenary (0.3.6)
  * mini_portile2 (2.0.0)
  * minitest (5.9.0)
  * multipart-post (2.0.0)
  * net-dns (0.8.0)
  * nokogiri (1.6.7.2)
  * octokit (4.3.0)
  * public_suffix (1.5.3)
  * rb-fsevent (0.9.7)
  * rb-inotify (0.9.7)
  * rouge (1.10.1)
  * safe_yaml (1.0.4)
  * sass (3.4.22)
  * sawyer (0.7.0)
  * terminal-table (1.5.2)
  * thread_safe (0.3.5)
  * typhoeus (0.8.0)
  * tzinfo (1.2.2)

**Comment by inkyfingers** [comment]
$ bundle exec jekyll serve --incremental;
jekyll 3.0.5 | Error:  Could not find a JavaScript runtime.
sudo apt-get install nodejs
$ bundle exec jekyll serve --incremental;
Server running... :+1:

**Comment by davelab6** [comment]
Nice! Since I expect to move from jekyll to grow.io for l10n I don't think its important to revise installation docs.

**Comment by quozl** [comment]
Closing because it will need to be rebased for #67.

**Pull request closed without merging**